src/types/events.rs

```diff
@@ -0,0 +1,147 @@
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct STXTransferEventData {
+    pub sender: String,
+    pub recipient: String,
+    pub amount: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct STXMintEventData {
+    pub recipient: String,
+    pub amount: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct STXLockEventData {
+    pub locked_amount: String,
+    pub unlock_height: String,
+    pub locked_address: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct STXBurnEventData {
+    pub sender: String,
+    pub amount: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct NFTTransferEventData {
+    #[serde(rename = "asset_identifier")]
+    pub asset_class_identifier: String,
+    #[serde(rename = "value")]
+    pub asset_identifier: String,
```

Comment:
The naming that we're using currently (and being corrected / abstracted in this PR) is source of confusion for developers. There was a discussion on the Stacks forum that I'm unable to retrieve.
This new types and schemas, based on Rosetta, are an opportunity to abstract and fix some confusing that we came up with a few months ago.

Reply:
I remember @friedger being involved in that discussion, maybe he can point us to it.

Reply:
Got it. What stops us from changing the field names in these structs to match?

Reply:
Changing the name of these fields at the stacks-node level would break clients like the stacks-blockchain-api, but also other clients that we don't have control over.